app/src/main/java/com/example/henzoshimada/feeltrip/EditMoodActivity.java

```diff
@@ -285,6 +288,13 @@ public void datePick(Context context) {
                 dateTime.get(Calendar.MONTH), dateTime.get(Calendar.DAY_OF_MONTH)).show();
     }
 
+    public void timePick(Context context) {
+        //selectDate();
+        Log.d("Mytag", "Went into date");
+        new TimePickerDialog(context, AlertDialog.THEME_DEVICE_DEFAULT_LIGHT, onTimeSetListener, dateTime.get(Calendar.HOUR_OF_DAY),
+                dateTime.get(Calendar.MINUTE), true).show();
+    }
+
```

Comment:
added time picker to let user pick a time, @forgeno needs to added a button and call:
timePick(findViewById(R.id.date_bottom_button).getContext());